examples/eight_schools/main_mcmc.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import, division, print_function
+
+import argparse
+import logging
+import sys
+
+import numpy as np
+import pandas as pd
+import torch
+
+import data
+import pyro
+import pyro.distributions as dist
+import pyro.poutine as poutine
+from pyro.infer import EmpiricalMarginal
+from pyro.infer.mcmc import MCMC, NUTS
+
+
+def model(sigma):
+    eta_dist = dist.Normal(torch.zeros(data.J), torch.ones(data.J))
+    mu_dist = dist.Normal(loc=torch.zeros(1), scale=10 * torch.ones(1))
```

Comment:
nit: you should use a consistent format for parameterizing these distributions. i recommend using positional args to be consistent with the other examples